src/index.html

```diff
@@ -2,13 +2,66 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css">
+    <link
+      class="header_img"
+      rel="stylesheet"
+      href="images/logo.png">
```

Comment:
Need to fix it everywhere
```suggestion
    <link
      class="header_img"
      rel="stylesheet"
      href="images/logo.png"
    >
```